### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
Updated EN -> JA documentation translations

English sentences exist in this PR. Please leave them as they are. (They will be translated next time.)

### CODE REVIEW

File: jekyll/_cci2_ja/architecture.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ toc::[]
 +
 WARNING: 移行サービスが起動時に停止している場合、接続先のサービスが動作しません。
 
-* Platinum サポートを利用している場合は、一部のサービスを外部化することができ (ここでは * を付けて示しています)、要件に合わせて管理できます。 外部化することで、データの安全性が高まり、システムに冗長性を持たせることができます。
+* Platinum サポートをご利用の場合は、一部のサービスを外部化することができ (ここでは * を付けて示しています)、要件に合わせて管理できます。 外部化することで、データの安全性が高まり、システムに冗長性を持たせることができます。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/architecture.adoc:25]: Not related to this PR, but
FYI, platinum is a legacy pricing plan name; this should say プレミアム　like the changes in other doc showed.

We should probably update this but I think the EN version needs to change too!

File: jekyll/_cci2_ja/aws.adoc [hunk 2]
```diff
@@ -193,7 +193,7 @@ WARNING: このページの設定はいつでも変更することができま
 
 . **Hostname:** ホスト名フィールドは、インストールプロセスで事前に入力されているはずですが、そのステップをスキップした場合は、Servies マシンのインスタンスのドメインまたはパブリック IP を入力してください。 [Test Hostname Resolution] をクリックすると、正確に入力されているか確認できます。
 
-. **Services ** : Services のセクションは、サービスを外部化する場合のみ使用します。 外部化は Platinum サービス契約のお客様のみご利用いただけます。 詳細については、support@circleci.com までお問い合わせください。
+. **Services ** : Services のセクションは、サービスを外部化する場合のみ使用します。 外部化はプレミアムサービス契約のお客様のみご利用いただけます。 詳細については、support@circleci.com までお問い合わせください。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/aws.adoc:196]: Interestingly, this says プレミアム here 🤔

File: jekyll/_cci2_ja/credits.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ version:
 チームに最適なプランを検討する際は、以下のような要素を考慮してください。
 
 - チームのユーザー数
-- 必要なサポートのレベル (コミュニティによるサポート、標準サポート、プレミアムサポート)
+- 必要なサポートのレベル (無料の Community サポート、Standard サポート、Premium サポート)
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/credits.md:21]: I don't know if we decide "premium" or プレミアム but it'll be good to make this consistent 🥸

File: jekyll/_cci2_ja/docker-layer-caching.md [hunk 4]
```diff
@@ -130,22 +130,22 @@ DLC が有効な場合、リモート ボリュームには `/var/lib/docker` 
...
-**注:** 以前、DLC は、`reusable: true` キーによって有効化されていましたが、 `reusable` キーは非推奨になり、`docker_layer_caching` キーがこれに代わりました。 `reusable` キーは非推奨になり、`docker_layer_caching` キーがこれに代わりました。 さらに、`exclusive: true` オプションも非推奨になり、すべてのリモート Docker VM が排他として扱われるようになりました。 つまり、DLC を使用すると、ジョブは必ず、他のジョブがアクセスできない排他的リモート Docker 環境を持つことになります。
+**注:** 以前、DLC は、`reusable: true` キーによって有効化されていましたが、 `reusable` キーは非推奨になり、`docker_layer_caching` キーがこれに代わりました。 さらに、`exclusive: true` オプションも非推奨になり、すべてのリモート Docker VM が排他として扱われるようになりました。 つまり、DLC を使用すると、ジョブは必ず、他のジョブがアクセスできない排他的リモート Docker 環境を持つことになります。
 
-### machine Executor
+### Machine Executor
 {: #machine-executor }
 {:.no_toc}
 
-Docker レイヤーキャッシュは、[`machine` Executor]({{ site.baseurl }}/ja/2.0/executor-types/#using-machine) を使用して Docker イメージをビルドする際のジョブ実行時間を短縮することもできます。 `machine` キーの下に `docker_layer_caching: true` を追加することで (後述の[例](#configyml)を参照)、`machine` Executor で DLC を使用できます。
+Docker レイヤーキャッシュは、[`maxhine` Executor]({{ site.baseurl }}/ja/2.0/executor-types/#using-machine) を使用して Docker イメージをビルドする際のジョブ実行時間を短縮することもできます。 `machine` キーの下に `docker_layer_caching: true` を追加することで (後述の[例](#configyml)を参照)、`machine` Executor で DLC を使用できます。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/docker-layer-caching.md:141]: machine

File: jekyll/_cci2_ja/getting-started.md [hunk 5]
```diff
@@ -18,52 +18,56 @@ CircleCI ではじめてビルドを成功 (グリーンビルド) させるた
...
 {: #digging-into-your-first-pipeline }
 
-ここまでの手順を終えると、自動的にパイプラインの実行が開始され、成功するのを確認できます。 実行結果を確認しましょう。 パイプラインの緑色の **[Success]** ボタンをクリックして、実行について以下の部分を確認しましょう。
+実行結果を確認します。
 
-1. **実行されたワークフローを確認する**: **[Success]** をクリックすると、実行されたジョブの一覧ページに移動します。 はじめてのビルドであれば、**1 つのジョブ** だけが実行されているはずです (**1 つのワークフロー**内で自動的に実行されます) 。  今回は、 `say-hello`という名前のジョブだけが実行されました。 `say-hello` をクリックして、ジョブのステップを調査してみましょう。
+1. プロジェクトのパイプラインのページで、緑色の **Success** ボタンをクリックすると、実行されたワークフローが表示されます (`say-hello-workflow`)。
+2. このワークフロー内で、パイプラインが `say-hello` というジョブを実行しました。 `say-hello` をクリックして、このジョブのステップを確認します:   
+   a. 環境のスピンアップ  
+   b.  環境変数の作成
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/getting-started.md:52]: looks like there may be a small space / indent issue for b.

File: jekyll/_cci2_ja/install-overview.adoc [hunk 6]
```diff
@@ -25,11 +25,11 @@ toc::[]
 
 === AWS 以外のプラットフォームのサポート
 
-Platinum サポートパッケージでは、Azure などお客様の組織でお使いのすべてのプラットフォームに  CircleCI をインストールおよび設定することが可能です。 まずは、https://support.circleci.com/hc/ja/requests/new[CircleCI サポート]またはお客様のアカウント担当者にお問い合わせください。
+Premium サポートパッケージでは、Azure などお客様の組織でお使いのすべてのプラットフォームに  CircleCI をインストールおよび設定することが可能です。 まずは、https://support.circleci.com/hc/ja/requests/new[CircleCI サポート]またはお客様のアカウント担当者にお問い合わせください。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/install-overview.adoc:28]: same topic! 😅

File: jekyll/_cci2_ja/ip-ranges.md [hunk 7]
```diff
@@ -160,7 +160,7 @@ IP アドレスの範囲機能が有効なジョブだけでなく、*すべて
 ## CircleCI macOS 用クラウド
 {: #circleci-macos-cloud }
 
-上記の AWS や GCP に加えて、CircleCI の macOS 用クラウドでもマシンが実行するジョブをホストしています。 The following IP address ranges are used by CircleCI macOS Cloud:
+上記の AWS や GCP に加えて、CircleCI の macOS 用クラウドでもマシンが実行するジョブをホストしています。 CiecleCI の macOS 用クラウドでは下記の IP アドレスを使用します。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/ip-ranges.md:163]:
```suggestion
上記の AWS や GCP に加えて、CircleCI の macOS 用クラウドでもマシンが実行するジョブをホストしています。 CircleCI の macOS 用クラウドでは下記の IP アドレスを使用します。
```

File: jekyll/_data/ja/sidenav.yml [hunk 8]
```diff
@@ -343,10 +345,11 @@ en:
           - 
             name: v2.19 Installation Guide
           - 
+          - 
           - 
             name: ユーザーアカウント
           - 
-            name: Orbs
+            name: Orb の管理
```
Comment (kelvintaywl) [inline jekyll/_data/ja/sidenav.yml:352]:
```suggestion
            name: Orbs の管理
```

**Review by kelvintaywl — COMMENTED** [review]
Mostly LGTM, besides some typos and mainly if Premium should be in English or プレミアム

File: jekyll/_cci2_ja/aws.adoc (hunk 2 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/aws.adoc:196]: Thank you, I'll go with Premium

File: jekyll/_cci2_ja/architecture.adoc (hunk 1 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/architecture.adoc:25]: Ah sorry, this is mistranslation! En version says "Premium". fixed!

**Review by kelvintaywl — APPROVED** [review]

**Pull request merged**